.github/workflows/build_monolith.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build Monolith
+on:
+  repository_dispatch:
```

Comment:
Can you add workflow_dispatch to this one also?  That way we can build on-demand